desktop/assets/texts/sv-SE/main_menu.ftl

```diff
@@ -13,6 +13,8 @@ file-menu-open-quick = Öppna...
 file-menu-open-advanced = Öppna Avancerat...
 file-menu-close = Stäng
 file-menu-reload = Ladda Om
+file-menu-recents = Nyliga
+file-menu-recents-empty = Inga nyliga inlägg
```

Comment:
Made note about this being sus on CrowdIn. Could any Swedish speakers confirm/refute?